Calendar/js/app.js

```diff
@@ -0,0 +1,233 @@
+var _date = new Date();
+var _month = _date.getMonth();
+var _year = _date.getFullYear();
+var _calendar = document.getElementById('calendar');
+var _selectedMonthElement = document.getElementById('select-month');
+var _selectedYearElement = document.getElementById('select-year');
+var _weekElement = document.getElementsByClassName('week')[0];
+var _startYear;
+var _endYear;
+var _months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+
+var birthdays = {
+    "1" : {
+        "3" : ["Shubham Katariya"],
+        "15" : ["Darshana Shrivastava", "Shikha Shakarwar", "Vipin Sir"],
+        "31" : ["Sagar Mishra"]
+    },
+    "2" : {
+        "" : []
+    },
+    "3" : {
+        "" : []
+    },
+    "4" : {
+        "2" : ["Gurpreet Chhabra"],
+        "22" : ["Sonam Ravi Gupta"],
+        "30" : ["Ram Bhaiya"]
+    },
+    "5" : {
+        "3" : ["Siyaram Patidar"],
+        "8" : ["Shubham Choubey"],
+        "9" : ["Mayur Vaidya"],
+        "10" : ["Amit Nagar", "Deepak Patidar"],
+        "28" : ["Rahul Kulmi"]
+    },
+    "6" : {
+        "20" : ["Vishal Patidar"]
+    },
+    "7" : {
+        "6" : ["Awanish Tiwari"],
+        "21" : ["Surendra Patidar"],
+        "24" : ["Anjana Singh"]
+    },
+    "8" : {
+        "8" : ["Aaditya Paliwal"]
+    },
+    "9" : {
+        "" : [""]
+    },
+    "10" : {
+        "16" : ["Piyush Chandak"],
+        "19" : ["Rashmi Soni"]
+    },
+    "11" : {
+        "19" : ["Priyanshi Asawara"]
+    },
+    "12" : {
+        "3" : ["Abhijeet Bhowmik"],
+        "7" : ["Gourav Agrawal"],
+        "11" : ["Shashank Saxena"],
+        "12" : ["Nitesh Thakur", "Satya Narayan patidar"]
+    }
+}
+
+function createCalendar() {
+    var startNumber = 1;
+    var lastDateOfPreviousMonth;
+    var totalRows;
+    var i = 1;
+    var j = 1;
+    var k = 1;
+    var dayOnFirstOfMonth = (new Date(_year, _month, 1)).getDay();
+    var lastDateOfCurrentMonth = (new Date(_year, _month + 1, 0)).getDate();
+    var lastDayOnLastDate = (new Date(_year, _month, lastDateOfCurrentMonth)).getDay();
+    var weekOnFirst = calculateWeekNo();
+    var weekOnLast;
+    console.log(weekOnFirst);
+    _calendar.innerHTML = "";
+    _weekElement.innerHTML = "";
+    _selectedMonthElement.value = _month;
+    _selectedYearElement.value = _year;
+
+
+    if (dayOnFirstOfMonth != 0) {
+        lastDateOfPreviousMonth = (new Date(_year, _month, 0)).getDate();
+        startNumber = lastDateOfPreviousMonth - dayOnFirstOfMonth + 1;
+    }
+
+    totalRows = Math.floor(((dayOnFirstOfMonth + lastDateOfCurrentMonth - 1) / 7) + 1);
+    weekOnLast = weekOnFirst + totalRows - 1;
+    dayOnFirstSaturday = 7 - dayOnFirstOfMonth;
+
+    for (var i = startNumber; i <= lastDateOfPreviousMonth; i++) {
+        _calendar.innerHTML += '<div class="dates prev-next-date"><span>' + i + '</span><div>';
+    }
+
+    for (var i = 1; i <= lastDateOfCurrentMonth; i++) {
```

Comment:
You can handle current date highlighting in this loop. so need of `showCurrentDate()`